coalib/bearlib/languages/definitions/DOT.py

```diff
@@ -0,0 +1,19 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class DOT:
+    aliases = 'DOT'
+    extensions = '.gv'
```

Comment:
Add a trailing comma in `aliases` and `extensions` as in other languages (to indicate it's a tuple).

Reply:
Done 👍